lhotse/bin/modes/recipes/librispeech.py

```diff
@@ -34,11 +34,18 @@
     default=1,
     help="How many threads to use (can give good speed-ups with slow disks).",
 )
+@click.option(
+    "--to-lowercase",
+    type=bool,
+    default=False,
+    help="Conversion of transcripts to lower-vase (originally in uppercase).",
+)
```

Comment:
@vesis84 most other recipes have an option called `--normalize-text` with choices such as "upper", "kaldi", etc. Maybe you can name this option the same for consistency?